bin/vault_sync_kv_credential.py

```diff
@@ -146,8 +159,11 @@ def stream_events(self, inputs, ew):
                         self._logger.critical("{0} Missing required field {1}, quitting".format(input_name, argument_name))
                         sys.exit(-1)
 
+                argument_value = input_config.get(argument_name)
+                if self._arguments[argument_name].get('data_type') == Argument.data_type_boolean:
+                    argument_value = True if argument_value.lower() in ["true", "yes", "t", "y", "1"] else False
                 # use .get(argument_name) to use the default of None if missing
-                setattr(self, argument_name, input_config.get(argument_name))
+                setattr(self, argument_name, argument_value)
```

Comment:
Because I now have a boolean data type being used, I'm parsing its content to determine if it should be stored as `True` or `False`.